Userland/Libraries/LibJS/Runtime/CanonicalIndex.h

```diff
@@ -6,7 +6,6 @@
 
 #pragma once
 
-#include <AK/Concepts.h>
```

Comment:
This is used.

Reply:
Oh right, `FloatingPoint`! You're right, I'll remove that. It compiles without any issue though.